packages/electrode-react-webapp/lib/index.html

```diff
@@ -13,5 +13,6 @@
 <div class="js-content">{{SSR_CONTENT}}</div>
 {{WEBAPP_BODY_BUNDLES}}
 <script>if (window.webappStart) webappStart();</script>
+<noscript><p>JavaScript is Disabled</p></noscript>
```

Comment:
how about something like, "sorry, this web page requires JavaScript to function properly.  Please enable JavaScript in your browser and reload".